### PR TITLE
Add bio for Azka

### LINKED ISSUE #9: Add a bio for Azka
- [x] Create a branch
- [x] Create a githubstudent.md file
- [x] Open a pull request
- [x] Get and receive feedback
- [x] Merge the pull request

### PR DESCRIPTION
**Pull request by islama1**
Closes #9 - that issue requests a bio for Azka, this pull request provides.

### CODE REVIEW

File: Azka.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Github Student Bio
+
+* Name: Azka
```
Comment (patann1) [inline Azka.md:3]: @islama1 can you add a little more detail? what's the last movie you watched?

File: Azka.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Github Student Bio
+
```
Comment (patann1) [inline Azka.md:2]: can you add the name of the movie here

**Comment by islama1** [comment]
@patann1 how does this look now?

**Comment by patann1** [comment]
looks good :smile:

**Pull request merged**